Modix.Bot/Modules/InfractionModule.cs

```diff
@@ -66,6 +66,7 @@ public InfractionModule(IModerationService moderationService, IUserService userS
             {
                 Id = infraction.Id,
                 Created = infraction.CreateAction.Created.ToUniversalTime().ToString("yyyy MMM dd"),
+                Author = $"{infraction.CreateAction.CreatedBy.Username}#{infraction.CreateAction.CreatedBy.Discriminator}",
```

Comment:
Minor, but we have an extension method for this, which would make sure this stays consistent with the other places where we show usernames:

```suggestion
                Author = infraction.CreateAction.CreatedBy.GetFullUsername(),
```